### PR TITLE
feat: add toleration

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +167,17 @@ participants:
...
   el_extra_labels: {}
 
+  # A list of tolerations that will be passed to the EL client container
+  # Only works with Kubernetes
+  # Example: el_tolerations:
+  # - key: "key"
+  #   operator: "Equal"
+  #   value: "value"
+  #   effect: "NoSchedule"
+  #   tolerationSeconds: 3600
```
Comment (h4ck3rk3y) [inline README.md:177]: should we snake case this to be Kurtosis compliant?

File: .github/tests/tolerations.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+participants:
+  - el_client_type: reth
+    cl_client_type: teku
+    cl_split_mode_enabled: true
+    cl_tolerations:
+      - key: "node-role.kubernetes.io/master1"
```
Comment (h4ck3rk3y) [inline .github/tests/tolerations.yaml:6]: is this actually executing as a test? we don't have a proper multi node k8s cluster do we?

**Review by h4ck3rk3y — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (barnabasbusa) [reply README.md:177]: this has to be kubernetes compliant no?

File: .github/tests/tolerations.yaml (hunk 2 above)
Comment (barnabasbusa) [reply .github/tests/tolerations.yaml:6]: no lol, its just something I've been locally testing out with. Thought might as well push it

Comment (h4ck3rk3y) [reply .github/tests/tolerations.yaml:6]: so without taints and tolerations i guess the test tests that it doesn't completely break docker!

all our k8s tests so far are on CircleCI

File: README.md (hunk 1 above)
Comment (h4ck3rk3y) [reply README.md:177]: haha, at the end its k8s compliant but `toleration_seconds` is more in line with the rest of the configuration than `tolerationSeconds`

File: .github/tests/tolerations.yaml (hunk 2 above)
Comment (barnabasbusa) [reply .github/tests/tolerations.yaml:6]: yeah docker just ignores this.

File: README.md (hunk 1 above)
Comment (barnabasbusa) [reply README.md]: Done

**Pull request merged**